### PR TITLE
fix the problem that files and wiki link doesn't show up on search 

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
Fix the problem that files and wiki link doesn't show up on search. Closes https://github.com/CenterForOpenScience/osf.io/issues/3827.

### CODE REVIEW

File: website/static/js/search.js [hunk 1]
```diff
@@ -46,8 +46,6 @@ var User = function(result){
     self.degree = result.degree;
     self.school = result.school;
     self.url = result.url;
-    self.wikiUrl = result.url+'wiki/';
```
Comment (sloria) [inline website/static/js/search.js]: Why was this removed? It looks like this variable is used in the template.

**Comment by sloria** [comment]
I'm trying to understand the issue being addressed and reproduce locally; which projects in search did not show the links?

**Comment by lbanner** [comment]
@jlcohoon has some noted in #3827

File: website/static/js/search.js (hunk 1 above)
Comment (chennan47) [inline website/static/js/search.js]: These variable are created under user model, but user don't have wikiUrl and fielUrl.

**Comment by sloria** [comment]
Yes, but the issue didn't specify how to reproduce the bug or why those projects didn't show the links.

I clarified this in person with Nan: projects with *no tags* were not showing the Wiki and Files links. This PR fixes a logical error in displaying those.

**Comment by sloria** [comment]
PR looks good.
